code/modules/reagents/chemistry/recipes/special.dm

```diff
@@ -278,9 +278,9 @@ GLOBAL_LIST_INIT(medicine_reagents, build_medicine_reagents())
 	switch(kind)
 		if(RNGCHEM_INPUT)
 			var/list/possible_ingredients = list()
-			for(var/datum/reagent/chemical in GLOB.medicine_reagents)
-				if(initial(chemical.chemical_flags) & REAGENT_CAN_BE_SYNTHESIZED)
-					possible_ingredients += chemical
+			for(var/datum/reagent/compound as anything in GLOB.medicine_reagents)
```

Comment:
As you can probably guess, GLOB.medicine reagents is filled with types and not instances